src/common/common.cpp

```diff
@@ -498,6 +498,70 @@ bool handleViKey(QKeyEvent *event, QObject *eventReceiver)
     return true;
 }
 
+bool handleEmacsKey(QKeyEvent *event, QObject *eventReceiver)
+{
+  int key = event->key();
+  Qt::KeyboardModifiers mods = event->modifiers();
+
+  switch ( key ) {
+  case Qt::Key_V:
+    if (mods & Qt::ControlModifier) {
+      key = Qt::Key_PageDown;
+      mods = mods & ~Qt::ControlModifier;
+    } else if (mods & Qt::AltModifier) {
+      key = Qt::Key_PageUp;
+      mods = mods & ~Qt::AltModifier;
+    } else {
+      return false;
+    }
+    break;
+  case Qt::Key_N:
+    if (mods & Qt::ControlModifier) {
+      key = Qt::Key_Down;
+      mods = mods & ~Qt::ControlModifier;
+      break;
+    }
+    return false;
+  case Qt::Key_P:
+    if (mods & Qt::ControlModifier) {
+      key = Qt::Key_Up;
+      mods = mods & ~Qt::ControlModifier;
+      break;
+    }
+    return false;
+  case Qt::Key_Comma:
+    if ((mods & Qt::ControlModifier) && (mods & Qt::ShiftModifier)) {
+      key = Qt::Key_Home;
+      mods = mods & ~(Qt::ShiftModifier || Qt::ControlModifier);
+    } else {
+      return false;
+    }
+  case Qt::Key_Stop:
```

Comment:
What key is this? Looks like special key on some keyboards to stop loading page (as opposed to `Qt::Key_Refresh`).